### ISSUE TITLE
Suche nach 2D Designer/Designerin

### ISSUE THREAD

**Issue by ghost** [BoundfoxStudios/community-project#372]
### Beschreibung

Die Suche am eigenen Discord Server [(Übersicht 2D Design / Grafiken)](https://discord.com/channels/454361389935493124/1118906060338634873) bliebt bisher erfolglos. Deshalb wurde der Entschluss gefasst auf anderen Plattformen die Stelle/Stellen auszuschreiben.

Hierfür wird ein aussagekräftiger Ausschreibungstext, wie wir es vom [eigenen Server](https://discord.com/channels/454361389935493124/852549634601517066) kennen, benötigt.

Dieses Issue wurde auf Wunsch von Manu erstellt und dient zum Einwurf von Webseiten/Plattformen als auch zur Ausarbeitung & zur Ideeneinbringung vom Ausschreibungstext.

**Comment by ghost** [comment]
Da es anscheinend nur sehr wenige Deutsche 2D Artist Communities gibt, werden wir unsere Stellenbeschreibung auf englischsprachigen Communities ausschreiben. (obwohl wir dezitiert jemanden mit Deutschkenntnisse suchen!)

Deshalb empfehle ich eine Einleitung / ein Vorwort auf Englisch zu forumilieren und den Rest der Ausschreibung auf Deutsch zu belassen.

**Comment by ghost** [comment]
Vorschlag "The Club" (Discord)
https://discord.com/channels/650728305020108805/650730571273732165

**Comment by ManuelRauber** [comment]
Hier kommt man leider nicht hin, wenn man nicht auf dem Discord-Server ist :(

**Comment by ghost** [comment]
> Hier kommt man leider nicht hin, wenn man nicht auf dem Discord-Server ist :(

stimmt, komisch was da Discord macht. Wie können wir sonst den Link machen? Ansonsten muss man halt die Suchfunktion verwenden, da scheint The Club als erster Treffer auf, wenn man es wie oben angibt.

**Comment by ghost** [comment]
LinkedIn ... Suchbegriffe: "Game Designer" / "Gamedesigner" / "Game Artist" etc

**Comment by ghost** [comment]
ArtStation ... Suchbegriffe: "Game UI" +Germany

ArtStation hat generell eine gute Suchfunktion und bietet Vorschauen wo man direkt den jeweiligen Stil sieht.
**Personen gefunden:**
https://www.artstation.com/isabelverheyen
https://www.artstation.com/vovelic
https://www.artstation.com/mingapur

**Comment by ManuelRauber** [comment]
Ich hab mal ein paar direkt via ArtStation angeschrieben. Mal schauen. :)
